src/main/java/com/epam/spring/homework3/service/repository/impl/InvoiceRepositoryImpl.java

```diff
@@ -0,0 +1,55 @@
+package com.epam.spring.homework3.service.repository.impl;
+
+import com.epam.spring.homework3.service.model.Invoice;
+import com.epam.spring.homework3.service.repository.InvoiceRepository;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Component;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@Component
```

Comment:
use @Repository annotation instead, it actually has no difference from @Component in high level, but it has better readability for reviewer